src/gui/main-styles.ts

```diff
@@ -300,7 +300,7 @@ styles.registerStyle("main", () => {
 			"margin-bottom": px(size.vpad_xs),
 		},
 		".mb-l": {
-			"margin-bottom": px(size.vpad_large),
+			"margin-bottom": px(size.vpad_large) + "!important",
```

Comment:
We should not add `!important` to these base/utility classes, it only makes sense for very specific classes like classes for specific components. If you need it in some place, please specify it inline or make a special class for your use case.